roles/deploy/tasks/main.yml

```diff
@@ -27,6 +27,11 @@
       repo: "{{ repo }}"
       dest: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sqlite dependencies
```

Comment:
I don't know why newfia worked - its Gemfile includes sqlite3. Unless the sqlite headers are installed on ubuntu by default? If we use `--without development test` then we need to update the Gemfile in all our projects so the Capistrano gems are not in the development group. Otherwise the initial bundle install will succeed but the cap deploy will fail.